doc/developers/testsuite.html

```diff
@@ -15,40 +15,28 @@
     <h1>The deal.II Testsuite</h1>
 
     <p>
-      The deal.II testsuite consists of two parts: 
-      <i>build tests</i> and the
-      <i>regression testsuite</i>. While the build tests
-      are used to check that the
-      library can be compiled on different systems and with different (versions
-      of) compilers, the regression tests are actually run and their output
-      compared with previously stored output files to verify that what
-      worked yesterday still works today. These two testsuites are
-      described below.
-    </p>
-
-    <p>
-      deal.II has a testsuite that has, at the time this article is written
-      (mid-2013), some 2,900 small programs (growing by roughly one per day)
-      that we run every time we make a change to make sure that no existing
-      functionality is broken. The expected output for every test is stored in
-      an <code>*.output</code>, and when you run a test you are notified if a
-      test produces different output. These days, every time we add a
-      significant piece of functionality, we add at least one new test to the
-      testsuite, and we also do so if we fix a bug, in both cases to make sure
-      that future changes do not break what we have just checked in. Machines
-      running tests send results
-      back home and these are then converted into
+      deal.II has a testsuite that has, at the time this article was last
+      updated (April 2015), some 3,600 small programs (growing by roughly
+      one per day) that we run every time we make a change to make sure
+      that no existing functionality is broken. The expected output for
+      every test is stored in an <code>*.output</code>, and when you run a
+      test you are notified if a test produces different output.
+
+      These days, every time we add a significant piece of functionality,
```

Comment:
In HTML, you'll need `</p> <p>` here. While you're there, say `...in an <code>.output</code> file.`